packages/splits-kit/src/constants/chains.ts

```diff
@@ -82,7 +87,7 @@ export const CHAIN_INFO: ChainInfo = {
     label: 'Goerli',
     logoUrl: '/networks/ethereum_logo.svg',
     rpcUrls: [
-      `https://eth-goerli.g.alchemy.com/v2/${process.env.STORYBOOK_GOERLI_ALCHEMY_API_KEY}`,
+      `https://eth-goerli.g.alchemy.com/v2/${process.env.STORYBOOK_ALCHEMY_API_KEY}`,
```

Comment:
shouldn't these rpc urls be defined in storybook, but not in splits kit? like they're just for our usage in storybook. i guess i'm a bit confused as to what's part of storybook and what's part of the splits-kit package that others are using.

Reply:
hm, yeah you're right, these are only used in the storybook, so I set this up a bit strange..

perhaps we just want separate constant mapping for these rpcUrls within the storybook directory, so these don't get included in the bundle?

should probably have a .env.example as well

Reply:
ok yea lemme do a little refactoring